apps/dashboard/src/fairness/App.tsx

```diff
@@ -36,4 +41,43 @@ export class App extends React.Component<IAppProps> {
     };
     return <FairnessWizard {...dashboardProps} />;
   }
+
+  private generateRandomMetrics = (
+    request: IMetricRequest,
+    abortSignal?: AbortSignal
+  ): Promise<IMetricResponse> => {
+    const binSize = _.max(request.binVector) || 0;
+    const bins: number[] = new Array(binSize + 1)
+      .fill(0)
+      .map(() => Math.random() / 3 + 0.33);
+    const binBounds: IBounds[] = bins.map((bin) => {
+      return {
+        lower: bin - Math.pow(Math.random() / 3, 2),
```

Comment:
nit: maybe move this repeating lower/upper computation to some small function?

Reply:
Thanks for the comment! Moved to small function!